src/angular/styles/_typography.scss

```
@@ -1498,7 +1509,7 @@ textarea {
 }
 
 // We need to position icons inside the buttons in a special way.
-:is(.sbb-button, .sbb-alt-button, .sbb-secondary-button, .sbb-ghost-button, .sbb-frameless-button, .sbb-link:where(.sbb-button-base))
+:is(.sbb-button, .sbb-alt-button, .sbb-secondary-button, .sbb-ghost-button, .sbb-frameless-button, .sbb-link:where(.sbb-button-base)):not(.sbb-icon-button)
```

To avoid more specifity, would it also be possible to use it like that:
```suggestion
:is(.sbb-button, .sbb-alt-button, .sbb-secondary-button, .sbb-ghost-button, .sbb-frameless-button, .sbb-link:where(.sbb-button-base)):where(:not(.sbb-icon-button))
```